README.md

```diff
@@ -47,6 +48,102 @@ programming resources.
 * [Flawed Gems](#flawed-gems)
 * [Managing processes](#managing-processes)
 
+## Testing
+### Rspec Unit Tests
+* <a name="unit-describe"></a>
+  There should be a `describe` block for each method. For class methods, the `describe` statement should 
+  be `.method_name`, and for instance methods `#method_name`.
+<sup>[[link](#unit-describe)]</sup>
+
+* <a name="describe-valid-scopes"></a>
+  Use a new `describe` block for any validations or scopes that are tested.
+<sup>[[link](#describe-valid-scopes)]</sup>
+
```

Comment:
New item:
Don't use `before(:each) do`.  It's extra, unnecessary typing.  `before do` is equivalent.